.github/workflows/github-actions-PR-validation.yml

```diff
@@ -54,6 +54,9 @@ jobs:
       - name: Package API
         run: |
           mvn clean package -Dquarkus.package.type=uber-jar -ntp
+      - name: Create test containers properties file
+        run: |
+          touch ~/.testcontainers.properties
```

Comment:
Looks like github now requires this file for test containers... thanks github for letting us know.